src/math/Math.sol

```diff
@@ -6,6 +6,24 @@ pragma solidity ^0.8.0;
 /// @custom:contact security@morpho.xyz
 /// @dev Library to perform simple math manipulations.
 library Math {
+    function abs(int256 x) internal pure returns (int256 y) {
```

Comment:
why not returning an `uint256`?

Reply:
but returning a `uint256` instead of an `int256` has several advantages:
- the function will always be exact and will never revert, even in the `type(int256).min` edge case.
- if the user needs a `uint256`, returning a `uint256` directly eliminates the conversion step, it improves efficiency since the default conversion from signed to unsigned integers performs a check on the sign, which is unnecessary in this case.
- if the user needs an `int256`, returning a `uint256` allows them to freely handle the edge case. They have the flexibility to perform the conversion at no additional cost if they know that the resulting amount will fit within the `int256` range. Alternatively, they can apply special logic in the edge case.

Also, `abs(type(int256).min) = 2 ** 255` which only fits in 256 bits.

Reply:
I agree that the function would be very clean if it returned an uint256, but I think that the assumption @Rubilmax made is sound and that there is a use case for staying in `int256`. If the user wanted to use `uint256` instead, then there are other functions to do that. For example the subtraction could be using the `zeroFloorSub` (from the current file).

Reply:
But now it seems both functions have equivalent advantages 😁 so I'd like to implement your suggestion: shall I name it `uabs` ?

Reply:
Maybe `unsignedAbs` ?